api/v1alpha1/grafanaorganization_types.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
These are some kubebuilder-generated files, right?
It comes as Apache License by default; Is Apache License what we want to use? Do we even know? (I don't :sweat_smile: )

Reply:
Let's go with yes for now https://github.com/giantswarm/object-storage-operator/blob/510134ef50102eb40c1fa0d7cd3a5e6bd6f8e03a/api/v1alpha1/bucket_types.go#L4 :D

Reply:
Honestly : I don't know :shrug:

Reply:
But it's a good question

Reply:
https://gigantic.slack.com/archives/CLPMFRVU6/p1728899214247859 Looks fine